tekton/ci/templates/kustomization.yaml

```diff
@@ -2,4 +2,6 @@ resources:
 - plumbing-template.yaml
 - all-template.yaml
 - tekton-cd-template.yaml
-- bindings.yaml
\ No newline at end of file
+- bindings.yaml
+- website-template.yaml
+- tekton-python-test.yaml
```

Comment:
`tekton-python-test` is under the `tekton/ci/job` folder so it should go into the `kustomize.yaml` in there instead.